src/main/java/org/recommendation/data/helper/UserDataHelper.java

```diff
@@ -0,0 +1,26 @@
+package org.recommendation.data.helper;
+
+import org.recommendation.model.User;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserDataHelper {
+    public static Map<String, User> userMap = new HashMap<>();
```

Comment:
why it is public? Shouldnt we have it private ?

Reply:
same as above